spec/requests/inventories_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'Inventories', type: :request do
+  describe 'GET /index' do
+    before :example do
+      get inventories_path
+    end
```

Comment:
- [ ] Since we are going to do the authentication after the controllers and this controller's actions need to be accessed by authenticated user, it would be good to add the devise `sign_in` `user` here to make the requests.